packages/wrangler/src/pages/functions/buildPlugin.ts

```diff
@@ -45,11 +45,10 @@ export function buildPluginFromFunctions({
 		minify,
 		sourcemap,
 		watch,
-		legacyNodeCompat,
 		// We don't currently have a mechanism for Plugins 'requiring' a specific compat date/flag,
 		// but if someone wants to publish a Plugin which does require this new `nodejs_compat` flag
 		// and they document that on their README.md, we should let them.
-		nodejsCompat: true,
+		nodejsCompatMode: nodejsCompatMode ?? "v1",
```

Comment:
does this comment still apply?

Reply:
Yes, basically what this does is set the nodejsCompat (aka v1) to true unless there is a mode provided. In this case there will never be a mode provided since there is still no way for the user to specify one.

Reply:
I could have just hard-coded it to v1 but this way, when we do finally provide such a mechanism, this will pick it up straight away.

Reply:
yeah that's what threw me off – I thought the mechanism had been provided since it tries using the var before defaulting to v1